controllers/cart/getCart.js

```diff
@@ -0,0 +1,34 @@
+const Cart = require('../../models/cart')
+const Store = require('../../models/store')
+const Product = require('../../models/product')
+
+async function getCart(req, res) {
+  try {
+    const storeId = req.params.store_id
```

Comment:
I don't think we'll be supplying any storeId, we'll be supplying only the cartId